docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+web:
+  build: .
+  command:
+    - python manage.py runserver 0.0.0.0:8000
```

Comment:
`command` is not a list. `command: python manage.py runserver 0.0.0.0:8000` without `-`